deeprankcore/domain/forcefield/__init__.py

```diff
@@ -58,17 +59,12 @@ def _find_matching_residue_class(self, residue: Residue):
         return None
 
     def get_vanderwaals_parameters(self, atom: Atom):
-        type_ = self._get_type(atom)
```

Comment:
Don't we need type_ var?

Reply:
It's used for getting the right vanderwaals parameters from the table file.

Reply:
As far as I could see it was only used internally by the `get_vanderwaals_parameters` function. However, this function basically did nothing else than read the type_ and get the parameters, so instead I merged `get_type` into `get_vanderwaals_parameters`.
I tested that old and new `add_features` functions (from atomic_contact, which uses this) give the exact same result on ~5 test cases, so am fairly confident that it is still being read correctly.